### PR TITLE
Add basic poetry installation instructions to README + .gitignore 

### PR DESCRIPTION
**Pull request by dekatzenel**
Please confirm these are correct and make sense

### CODE REVIEW

**Comment by debugmiller** [comment]
Poetry is not needed. A user can do:
```
pip -m venv venv
source venv/bin/activate
pip install pypanther
```

**Comment by dekatzenel** [comment]
> Poetry is not needed. A user can do:
> 
> ```
> pip -m venv venv
> source venv/bin/activate
> pip install pypanther
> ```

I think this should be `pip install .` to get all of the dependencies, but otherwise looks reasonable

**Review by kostaspap — APPROVED** [review]
LGTM, I also struggled a bit with this in the beginning

File: README.md [hunk 1]
```diff
@@ -12,6 +12,9 @@ Before you begin, make sure you have the following installed:
 - Git
 - Make
 - Python (check the `pyproject.toml` file for the required version)
+- Poetry
```
Comment (kostaspap) [inline README.md:15]: per @debugmiller 's comment, it seems poetry is not really needed but it's one of the options?

Comment (dekatzenel) [reply README.md:15]: True, but see [this slack comment](https://panther-labs.slack.com/archives/C06VCSMHZE1/p1720543884338609?thread_ts=1720464728.444789&cid=C06VCSMHZE1)

Comment (kostaspap) [reply README.md:15]: Ah! Makes sense to me :)

**Pull request merged**